parl/remote/client.py

```diff
@@ -283,6 +287,7 @@ def connect(master_address):
 
     Args:
         master_address (str): The address of the Master node to connect to.
+        distributed_files (list): A list of files to be sent to the job.
```

Comment:
A list of files to be distributed at all remote instances(e,g. the configuration file for initialization) .